ezsystems/ezplatform-query-fieldtype
# EZP-31556: Registered Field Type as not to be indexed by search engine

| Question                                  | Answer
| ---------------------------------------- | ------------------
| **JIRA issue**                          | [EZP-31556](https://jira.ez.no/browse/EZP-31556)
| **Type**                                   | bug
| **Target eZ Platform version** | eZ Platform `v2.5+`
| **BC breaks**                          | no
| **Tests pass**                          | yes
| **Doc needed**                       | no

It's a back-port of #33 for `1.0` branch related to eZ Platform 2.5

Query Field Type is not searchable so there's no reason it should produce Search Index data. On 2.5 it only clutters Solr search index, does not crash.

Obsolete `SearchField` has been dropped (no BC promise for that, this is not an extensibility point).

#### QA

The bug is not visible[1] for eZ Platform 2.5, so only sanity check for reindexing when Content with Query Field Type is used.

_[1] Well, ok, you could check Solr Index data to see that `<your_field_def>_<your_ct_identifier>_value_s` field is gone from Solr Index for a given Content item, but not sure if this is worth your time_

#### TODO:

- [x] Prepare merge up PR for 3.0 (#35).
